hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/selectors/CustomSelector.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.spec.infrastructure.selectors;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.BiFunction;
+
+public class CustomSelector implements BiFunction<Set<String>, Set<String>, Optional<String>> {
```

Comment:
Remove, see https://github.com/hashgraph/hedera-services/pull/12123#discussion_r1532143841

Reply:
Comment addressed. Now instead of CustomSelector I'am using the new RandomSelector(account -> account.equals(HOLLOW_ACCOUNT)). Deleted the CustomSelector since it is not used now.